File: app/src/main/java/com/mecong/tenderalarm/TenderAlarmApp.java
package com.mecong.tenderalarm;

import android.app.Application;

import timber.log.Timber;

public class TenderAlarmApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        } else {
            Timber.plant(new TimberReleaseTree());
        }
    }
}

File: app/src/main/java/com/mecong/tenderalarm/TenderAlarmApp.kt
package com.mecong.tenderalarm

import android.app.Application
import timber.log.Timber

class TenderAlarmApp : Application() {
    override fun onCreate() {
        super.onCreate()

        if (BuildConfig.DEBUG) {
            Timber.plant(TimberDebugTree(this.applicationContext))
        } else {
            Timber.plant(TimberReleaseTree())
        }
    }
}